event/main_listener.php

```diff
@@ -215,6 +220,23 @@ public function disable_media_embed($event)
 		}
 	}
 
+	/**
+	 * Setup a cache directory to improve scraping performance
+	 *
+	 * @param \phpbb\event\data $event The event object
+	 */
+	public function setup_cache_dir($event)
+	{
+		/** @var \phpbb\textformatter\s9e\parser $service  */
+		$service = $event['parser'];
+		$parser = $service->get_parser();
+
+		if (!$this->disable_plugin && $this->cache_dir)
```

Comment:
You do not need to check disable_plugin. The plugin is never actually disabled. These are just confusing language for either 1) disabling the rendering of plain URLs or 2) stopping a user from posting with the MEDIA bbcode.